core/src/com/po/fuck/view/classdrawers/factories/EntityDrawerFactory.java

```diff
@@ -17,8 +17,10 @@
 public class EntityDrawerFactory {
     
     static {
-        Renderer.addDrawer(Player.class, EntityDrawerFactory.get(Player.class, getAnimation("FUCKerWithoutHands2.png")));
-        Renderer.addDrawer(BasicEnemy.class, EntityDrawerFactory.get(BasicEnemy.class, getAnimation("player2.png")));
+        Renderer.addDrawer(Player.class, EntityDrawerFactory.get(Player.class,
+                getAnimation(Player.class)));
+        Renderer.addDrawer(BasicEnemy.class, EntityDrawerFactory.get(BasicEnemy.class,
+                getAnimation(BasicEnemy.class)));
     }
 
     public static <T extends Entity> ObjectDrawer<T> get(Class<T> clz, Animation<TextureRegion> animation){
```

Comment:
Can't we not pass `Animation<TextureRegion> animation` and use `getAnimation(BasicEnemy.class)` there ?

Reply:
We can